frontend/src/components/VideoQualityLabel/VideoQualityLabel.types.ts

```diff
@@ -0,0 +1,8 @@
+interface VideoQualityLabelProps {
+    /** The type of video quality label to display 
```

Comment:
there is no need in this comments, type description says us what exactly `type` could be